projects/packages/account-protection/src/class-password-detection.php

```diff
@@ -402,6 +402,12 @@ private function user_requires_protection( \WP_User $user, string $password ): b
 			return false;
 		}
 
+		$user_requires_protection = apply_filters( Config::TRANSIENT_PREFIX . '_user_requires_protection', true, $user );
```

Comment:
```suggestion
		/**
		 * Filter which determines whether or not account protection should be applied for the provided user.
		 *
		 * @since $$next-version$$
		 *
		 * @param bool     $requires_protection Whether or not account protection should be applied.
		 * @param \WP_User $user                The user object to apply the filter against.
		 */
		$user_requires_protection = apply_filters( Config::TRANSIENT_PREFIX . '_user_requires_protection', true, $user );
```